meta/contributing.md

```diff
@@ -1,6 +1,99 @@
 # Contributing
 
-TODO
+Documentation is critical to making a usable data platform.
+When surveying our users,
+their most common complain has been our lack of documentation.
+It's important that we improve our documentation as often as possible.
 
-https://github.com/mozilla/telemetry-onboarding/blob/master/Contributing.md
+## Getting the Raw Documentation
 
+The documentation is intended to be read as HTML at 
+[docs.telemetry.mozilla.org](https://docs.telemetry.mozilla.org).
+However, we store the documentation in raw text files in the
+[firefox-data-docs repo](https://github.com/mozilla/firefox-data-docs).
+To begin contributing to the docs, fork the firefox-data-docs repo.
+
+## Building the Documentation
+
+The documentation is rendered with [Gitbook](https://www.gitbook.com/)
+To build the documentation locally,
+you'll need to install the gitbook-cli app with npm:
```

Comment:
consider linking `npm` to the npm installation instructions: https://www.npmjs.com/get-npm

Reply:
done